string topic    # Name of the topic to connect
---
bool success    # indicate successful connection
string message  # informational, e.g. for error messages